__tests__/server.test.js

```diff
@@ -105,6 +105,34 @@ describe("GET /api/articles", () => {
         expect(body.articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("should return only the articles that have the correct topics according to the query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBeGreaterThan(0);
```

Comment:
You can be a bit more specific here, based on the test data how many articles should there be in the response ? - that is the length we would expect